dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/log/remote/RemoteLogConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.log.remote;
+
+import static org.apache.dolphinscheduler.common.constants.Constants.REMOTE_LOGGING_YAML_PATH;
+
+import org.apache.dolphinscheduler.common.config.IYamlDelegate;
+import org.apache.dolphinscheduler.common.config.ImmutableYamlDelegate;
+import org.apache.dolphinscheduler.common.constants.Constants;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+
+@UtilityClass
+@Slf4j
+public class RemoteLogConfiguration {
+
+    private static final IYamlDelegate yamlDelegate =
+            new ImmutableYamlDelegate(REMOTE_LOGGING_YAML_PATH);
+
+    public static Boolean isEnable() {
+        return yamlDelegate.getBoolean(Constants.REMOTE_LOGGING_ENABLE, Boolean.FALSE);
+    }
+
+    public static String getTarget() {
+        return yamlDelegate.getString(Constants.REMOTE_LOGGING_TARGET).toUpperCase();
+    }
+
+    public static String getLogBaseDir() {
+        return yamlDelegate.getString(Constants.REMOTE_LOGGING_BASE_DIR);
+    }
+
+    public static Integer getThreadPoolSize() {
+        return yamlDelegate.getInt(Constants.REMOTE_LOGGING_THREAD_POOL_SIZE, 10);
+    }
+
+    public static IYamlDelegate getYamlDelegate(String key) {
+        return yamlDelegate.getYamlDelegate(key);
+    }
```

Comment:
The delegate shouldn't exposed.

Reply:
same reason as https://github.com/apache/dolphinscheduler/pull/15826#discussion_r1565238503